src/courtroom.cpp

```diff
@@ -1372,6 +1370,16 @@ void Courtroom::done_received()
 
   show();
 
+  if (Options::getInstance().restoreWindowPositionEnabled())
+  {
+    auto maybe_point = Options::getInstance().windowPosition("courtroom");
+    if (maybe_point.has_value() && ao_app->pointExistsOnScreen(maybe_point.value()))
+    {
+      qDebug() << maybe_point.value();
+      QMainWindow::move(maybe_point.value());
+    }
+  }
```

Comment:
The position should be restored on creation of the courtroom, within AOApplication, not from within the Courtroom.

Additionally, it should be consistent with the Lobby, centering if required to the primary screen, no reason it shouldn't here.